src/util.ts

```diff
@@ -2,13 +2,13 @@ export function isAPromise(obj: any): boolean {
   return obj.then && typeof obj.then === 'function'
 }
 
-export function quoteAndCombine(...args) {
+export function quoteAndCombine(...args: (string | number | boolean | any[])[]) {
   if (Array.isArray(args[0])) {
     args = args[0]
   }
 
   return args
-    .filter(arg => arg)
-    .map(arg => `'${arg}'`)
+    .filter((arg) => arg === '' ? `''` : arg)
```

Comment:
Because `.filter()` just needs to return a boolean, you can make this line just

```ts
.filter(arg => arg === '' || arg)
```

Reply:
Done